### ISSUE TITLE
Question about label-free claim: why is kd_gamma=1.0 used in the provided scripts?

### ISSUE THREAD

**Issue by NoraAKA** [lcdbsa522/SQuaT#1]
Thank you for your excellent work on SQuaT! I'm trying to reproduce your results and have a question about the label-free claim in your paper.

The paper states that SQuaT is a "label-free QAT framework with KD" that works "when training labels are often unavailable due to privacy, copyright, or cost constraints."

However, in the provided training script `run_cifar100_resnet32_squat.sh`, I noticed that `kd_gamma=1.0` is set, which enables the classification loss (`loss_cls`) that requires ground-truth labels:

python train_quant_distill.py \
    --kd_gamma 1.0 \     # Enables label-dependent loss
    --kd_beta 1.0 \      # KL divergence loss
    --distill 'fd'       # Feature distillation

Could you please clarify:
1. Does SQuaT require labels for the reported results?
2. If yes, what does "label-free" refer to in the paper?

Thank you!

**Comment by NoraAKA** [comment]
Additionally, I'm using the default config and observed two issues:

**1. On DeiT:** Training loss decreases normally (1.98 → 0.074), but eval loss stays around 4.48 and top-1 accuracy is only 3-6% (near random). The feature loss (key innovation) seems to hurt generalization.

**2. On ResNet32 (with default config):** Removing the feature loss occasionally gives better performance, suggesting the current implementation may be problematic.

**DeiT log (default config):**
| epoch | train_loss | eval_loss | eval_top1 |
|-------|------------|-----------|-----------|
| 0     | 1.982      | 4.903     | 0.00      |
| 50    | 0.127      | 4.484     | 4.92      |
| 100   | 0.108      | 4.483     | 5.45      |
| 150   | 0.097      | 4.479     | 5.36      |
| 200   | 0.088      | 4.488     | 4.56      |
| 250   | 0.080      | 4.494     | 4.20      |
| 299   | 0.074      | 4.481     | 4.53      |

**Questions:**
1. Which config might be causing this? How should I debug the feature loss?
2. In the paper, is the reported accuracy the last epoch or the best epoch?
